### PR TITLE
remove debug message in stderr, modify usage message, and add .gitignore

### PR DESCRIPTION
**Pull request by rkajitani**

### CODE REVIEW

File: src/pairedDBG.cpp [hunk 1]
```diff
@@ -11401,13 +11361,6 @@ void PairedDBG::consensusScaffolding(void)
             candidateIterator = candidate.erase(candidateIterator + minCandidateID);
         }
 
-//		cerr << "sort include" << endl;
-        std::sort(include.begin(), include.end());
```
Comment (ShunOuchi) [inline src/pairedDBG.cpp:11405]: This is nesecceary for correct consensus scaffolding.

File: src/pairedDBG.cpp [hunk 2]
```diff
@@ -12247,40 +12153,6 @@ void PairedDBG::checkLinkBetweenHiCNodePair(vector<ConsensusPart> scaffold1, vec
...
-            std::cerr << "\tgap:" << std::accumulate(gapDistribution[i].begin(), gapDistribution[i].end(), 0L) / static_cast<long>(gapDistribution[i].size());
-        std::cerr << std::endl;
-        //for (long j = 0; j < gapDistribution[i].size(); ++j) {
-        //    std::cerr << gapDistribution[i][j] << ", ";
-        //}
-        //if (gapDistribution[i].size() > 0)
-        //    std::cerr << std::endl;
-        std::cerr << "numLink(ignore insertLength):" << insDistribution[i].size();
-        if (insDistribution[i].size() != 0) {
-            std::sort(insDistribution[i].begin(), insDistribution[i].end(), [](const std::pair<long, long> &left, const std::pair<long, long> &right){return left.first < right.first;});
```
Comment (ShunOuchi) [inline src/pairedDBG.cpp:12271]: This sort is necessary for proper checking.

File: src/pairedDBG.cpp [hunk 3]
```diff
@@ -12247,40 +12153,6 @@ void PairedDBG::checkLinkBetweenHiCNodePair(vector<ConsensusPart> scaffold1, vec
         }
     }
```
Comment (ShunOuchi) [inline src/pairedDBG.cpp:12155]:
```suggestion
    for (long i = 0; i < 4; ++i) {
        if (insDistribution[i].size() != 0) {
             std::sort(insDistribution[i].begin(), insDistribution[i].end(), [](const std::pair<long, long> &left, const std::pair<long, long> &right){return left.first < right.first;});
        }
    }
```

**Comment by rkajitani** [comment]
I have updated the file.

**Pull request merged**